src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,8 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
```

Comment:
you can create styles file for `UserInfo` component and for other components as well